core/src/economic_viability.rs

```diff
@@ -218,12 +232,18 @@ mod tests {
                 ..Default::default()
             }))
         });
-        let economic_viability =
-            EconomicViabilityComputer::new(Arc::new(price_oracle), Arc::new(gas_station), 10.0);
+        let subsidy = 10.0f64;
+        let min_avg_fee_factor = 1.0f64;
```

Comment:
nit: I don't like using a `min_avg_fee_factor` of `1.0`, since theoretically this test would have passed before the modification to the `EconomicViabilityComputing::min_avg_fee` modification.

Reply:
Thanks. This was meant to be 1.1.